galpy/potential_src/Potential.py

```diff
@@ -1510,6 +1510,166 @@ def nemo_accpars(self,vo,ro):
         except AttributeError:
             raise AttributeError('NEMO acceleration parameters not supported for %s' % self.__class__.__name__)
 
+    @potential_physical_input
+    @physical_conversion('position',pop=True)
+    def rtide(self,R,z,M=None,M_units=None):
+        """
+            
+        NAME:
+            
+            rtide
+            
+        PURPOSE:
+            
+            Calculate the tidal radius for object of mass M assuming a circular orbit 
+            at R in potential Pot given the formalism of Bertin, G., & Varri, A. L. 2008, ApJ, 689, 1005
+        INPUT:
+            
+            Pot - Potential instance or list of such instances
+            
+            R - Galactocentric radius (can be Quantity)
+            
+            z - height (can be Quantity)
+            
+            M - (default = None) Mass of object (can be Quantity)
+            
+        OUTPUT:
+            
+            Tidal Radius
+        
+        HISTORY:
+            
+            2018-02-12 - Written - Webb (UofT)
+            
+        """
+        if self.isNonAxi:
+            #Make sure potential is axisymmetric
+            print("Error - cannot calculate rtide for non-axisymmetric potential")
+            return 0.0
+        elif M==None:
+            #If units not given, assume galpy! GM units?
+            print("Error - mass parameter has not been set")
+            return 0.0
+        else:
```

Comment:
We should just assume the following two cases:

1. M is a scalar --> assumed to be in galpy units --> divide by bovy_conversion.mass_in_msol
2. M is a Quantity --> try units of mass or Gxmass

So I think this entire piece of code should be
```
### NO ELSE TO BEGIN
if _APY_LOADED and isinstance(M,units.Quantity):
    try:
        M= M.to(units.Msun).value/bovy_conversion.mass_in_msol(self._vo,self._ro)
    except units.UnitConversionError: pass
    else:
        try:
            M= M.to(units.pc*units.km**2/units.s**2).value/bovy_conversion.mass_in_msol(self._vo,self._ro)/bovy_conversion._G
        except units.UnitConversionError: pass
            raise units.UnitConversionError('M= parameter of rtide should have units of mass')
```